weldx/tags/core/transformations/local_coordinate_system.py

```diff
@@ -7,8 +8,7 @@
 class LocalCoordinateSystemConverter(WeldxConverter):
     """Serialization class for weldx.transformations.LocalCoordinateSystem"""
 
-    name = "core/transformations/local_coordinate_system"
-    version = "0.1.0"
+    tags = [WELDX_TAG_URI_BASE + "core/transformations/local_coordinate_system-0.1.*"]
```

Comment:
Why now use this pattern?

Reply:
the old format was a legacy asdf style that is now superseded for full tag definitions. It is not really useful when mixing different tags/versions which is why I want to get away form it (the `*` syntax makes it easier now anyway)

In case you were referring to the use of `WELDX_TAG_URI_BASE`, that was purely due to line length here (I am not sure we should actually do that)

Reply:
I was referring to TAG_URI_BASE pattern. It looks like something special compared to the other converters, but it is not. So I'd suggest just to wrap the long line then.